rocky/rocky/templates/partials/form/field_input.html

```diff
@@ -1,13 +1,14 @@
 {% load i18n %}
 
 <div {% if field.field.required %}class="required"{% endif %}>
-    {{ field.label_tag }}
+    {% if field.field.required %}
+        <span>{{ field.label_tag }} <span class="nota-bene" aria-hidden>{% translate "(Required)" %}</span></span>
+    {% else %}
+        {{ field.label_tag }}
+    {% endif %}
```

Comment:
@HeleenSG can you validate this?

Reply:
I would recommend using a label instead of the parent span. That way you will get 

<label> 
   Label content (name)
   <span>required</span>
</label>

instead of a nested span.

Reply:
I've changed it!